### DISCUSSION TITLE
can we have windows version please?

### DISCUSSION THREAD

**Discussion by aresstokrat** [martomi/chiadog#108] [General]
python script to work with windows path's and processes? possible?

**Comment by pieterhelsen** [comment]
Windows support is available already. Instructions are currently only on the dev branch but will be released to main soon. 
https://github.com/martomi/chiadog/blob/dev/WINDOWS.md

**Reply by MadCoderOne332** [reply]
new instructions are the BOMB, I was up and running quickly. Windows 10x64 THANKS
